test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,21 @@
+require "test_helper"
+
+describe VotesController do
+  it "should create a vote" do
+    proc {
+      post votes_path, params: { vote: {user_id: 2, work_id: 9 }}
```

Comment:
Instead of adding in `user_id` into `params[:vote][:user_id]`, you need to pass it in through session. To get user_id into session, before line 5 add the following line:
```
    post '/login', params: {username: users(:bob).username}
```
and then modify this line (line 6) to the following
```
      post votes_path, params: { vote: {work_id: 9 }}
```